.github/workflows/ai-codereview.yml

```diff
@@ -48,14 +69,19 @@ jobs:
                 --lang "${LANG}" \
                 --template-file "${PATH_TO_JINJA_TEMPLATE_FILE}"
           done
-          echo "review_result=$(cat ./message.txt | tr '\n' ' ')" >> $GITHUB_OUTPUT
+          echo "See the code review here ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}" >> ./summary.txt
```

Comment:
nit: I'd suggest adding a link in the markdown format to beautify the comment: `See the code review [here](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})`